### PR TITLE
Sync bluetooth mapper from KubeEdge repo

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,12 +2,53 @@ module github.com/kubeedge/mappers-go
 
 go 1.14
 
+replace (
+	github.com/apache/servicecomb-kie v0.1.0 => github.com/apache/servicecomb-kie v0.0.0-20190905062319-5ee098c8886f
```
Comment (fisherxu) [inline go.mod:6]: This vendor can be deleted.

Comment (daixiang0) [reply go.mod:6]: Mappers-go import kubeedge device CRD, now kubeedge still need this vendor, cannot remove it.

**Comment by sailorvii** [comment]
Thanks for porting this module.
Now the bluetooth and its e2e test don't work well and need to be refined. The bluetooth configmap is changed and neet to fit to version 1.4/1.5. The e2e test will destroy the host environment after running.
A note about those would be helpful for the customer.

**Comment by sailorvii** [comment]
Please help to renew the copyright year to 2021.

**Comment by daixiang0** [comment]
The copyright year does not need to renew since we do not create new files.

@sailorvii thanks for input, this pr focuses on mapper move, let's fix them in other prs.

**Comment by daixiang0** [comment]
ping @kevin-wangzefeng

**Comment by kevin-wangzefeng** [comment]
Manually resolved conflict and merged, closing.

/close

**Pull request closed without merging**